Library/Formula/sip.rb

```diff
@@ -2,8 +2,8 @@
 
 class Sip < Formula
   homepage 'http://www.riverbankcomputing.co.uk/software/sip'
-  url "https://downloads.sf.net/project/pyqt/sip/sip-4.16.3/sip-4.16.3.tar.gz"
-  sha1 "7c4079d164ccbfe4a5274eaeebe8e3cc86e3a75a"
+  url "https://downloads.sf.net/project/pyqt/sip/sip-4.16.5/sip-4.16.5.tar.gz"
+  sha1 "d5d7b6765de8634eccf48a250dbd915f01b2a771"
   revision 1
```

Comment:
Can you remove this revision line? We don't need it after increasing the version. Thanks!